src/amo/components/SearchResult/styles.scss

```diff
@@ -32,6 +32,9 @@ $icon-default-size: 32px;
   &:hover {
     color: $link-color;
   }
+
+  // This margin gives the recommended badge some space.
+  @include margin-end(8px);
```

Comment:
Can we move this mixin at the top of the declaration? Or does it need to be at the end?

Reply:
Sure, I can move it